charts/controller/templates/controller-ingress-rule-controller.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.global.experimental_native_ingress }}
```

Comment:
Indeed, global variables are documented here:
https://github.com/kubernetes/helm/blob/master/docs/charts.md#global-values

In short, with normal variables child charts see variables set by parent chart, but without the namespace. So for example `.Values.foo.bar` is seen as `.Values.bar` from `foo` subchart.

The `global` namespace is special, because it's not stripped when passing variables to child subcharts, also:

> If a subchart declares a global variable, that global will be passed downward (to the subchart's subcharts), but not upward to the parent chart. There is no way for a subchart to influence the values of the parent chart.